internal/config/config.go

```diff
@@ -85,6 +86,10 @@ type TypeConfig struct {
 	SkipTypeCreate bool `yaml:"skip_type_create,omitempty"`
 }
 
+type QueryConfig struct {
+	Path string `yaml:"path"`
+}
```

Comment:
Oh nice, I was mocking something up yesterday too and came up with this.

```
+// Query is the information necessary to build a query method.  The Paths reference the the place in the hierarcy, while the names reference the objects within those paths to query.
+type Query struct {
+       // Path is the path of TypeNames in GraphQL that preceed the objects being queried.
+       Path []string `yaml:"path,omitempty"`
+       // Names is a list of TypeName entries that will be found at the above Path.
+       Names []string `yaml:"names,omitempty"`
 }
```

That seems like enough information to be able to specify where in the tree something is located, but also which items will be found at that particular level of the tree.  What do you think?

Reply:
Then the config would be something like this:
```
    queries:
      - path: ["actor"]
        names: ["entities", "entity", "entitySearch"]
```